### PR TITLE
Fix wrong redis key

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

- Fixes an issue where registry keys were created in a wrong manner
- Improves Pylint testing run by pytest. It will now display some meaningful error upon fail
- Refactors pylint testing utils, which now get imported from the common **pytest-simcore** library

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: packages/service-library/tests/test_package.py [hunk 1]
```diff
@@ -20,12 +20,13 @@ def pylintrc(osparc_simcore_root_dir):
...
-        assert subprocess.check_call(cmd) == 0
-    except subprocess.CalledProcessError as err:
-        pytest.fail("Linting error. Linter existed with code %d" % err.returncode)
+    AUTODETECT = 0
+    cmd = f"pylint --jobs={AUTODETECT} --rcfile {pylintrc} -v {package_dir}".split()
+    pipes = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    std_out, _ = pipes.communicate()
+    if pipes.returncode != 0:
+        print(std_out.decode('utf-8'))
+        assert False, "Pylint failed with error, check this test's stdout to fix it"
```
Comment (sanderegg) [inline packages/service-library/tests/test_package.py:29]: very nice, looking forward to see how this looks like

File: services/catalog/tests/unit/test_package.py [hunk 2]
```diff
@@ -23,7 +23,11 @@ def pylintrc(project_slug_dir, osparc_simcore_root_dir):
 
 def test_run_pylint(pylintrc, package_dir):
     cmd = "pylint --jobs 0 --rcfile {} -v {}".format(pylintrc, package_dir)
-    assert subprocess.check_call(cmd.split()) == 0
+    pipes = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (pcrespov) [inline services/catalog/tests/unit/test_package.py:26]: perhaps you can include this code in helpers within ``packages/pytest-simcore``

Comment (pcrespov) [reply services/catalog/tests/unit/test_package.py:26]: if you have a chance, update as well it in https://github.com/ITISFoundation/cookiecutter-simcore-pyservice as well

Comment (GitHK) [reply services/catalog/tests/unit/test_package.py]: > perhaps you can include this code in helpers within `packages/pytest-simcore`

Trying to add this at the moment. When I'm done please inspect how the new packages get installed.

File: packages/pytest-simcore/src/pytest_simcore/helpers/utils_pylint.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+""" Helps with running Pylint tests on different modules """
+import subprocess
+
+
+def assert_pylint_is_passing(pylintrc, package_dir, autodetect=0):
```
Comment (pcrespov) [inline packages/pytest-simcore/src/pytest_simcore/helpers/utils_pylint.py:5]: MINOR: ``autodect=0`` is gives the impression to be a flag to enable/disable "autodectection" which is not the case. It denotes the number of parallel jobs in pylint and it happens that "0" indicates auto-detection.  I would do something like

```python
AUTODETECT = 0

def assert_pylint_is_passing(pylintrc, package_dir, *, number_of_jobs: int = AUTODECT):
   ...

```

Comment (GitHK) [reply packages/pytest-simcore/src/pytest_simcore/helpers/utils_pylint.py:5]: why add the `*`  to it?

Comment (pcrespov) [reply packages/pytest-simcore/src/pytest_simcore/helpers/utils_pylint.py:5]: what do you mean? I like having options as enforced key-arguments

File: services/catalog/tests/unit/test_package.py (hunk 2 above)
Comment (GitHK) [reply services/catalog/tests/unit/test_package.py]: > if you have a chance, update as well it in https://github.com/ITISFoundation/cookiecutter-simcore-pyservice as well

also trying to apply the same changes here

**Pull request merged**